src/main/java/com/crowdin/client/fields/FieldsApi.java

```diff
@@ -0,0 +1,98 @@
+package com.crowdin.client.fields;
+
+import com.crowdin.client.core.CrowdinApi;
+import com.crowdin.client.core.http.HttpRequestConfig;
+import com.crowdin.client.core.http.exceptions.HttpBadRequestException;
+import com.crowdin.client.core.http.exceptions.HttpException;
+import com.crowdin.client.core.model.ClientConfig;
+import com.crowdin.client.core.model.Credentials;
+import com.crowdin.client.core.model.PatchRequest;
+import com.crowdin.client.core.model.ResponseList;
+import com.crowdin.client.core.model.ResponseObject;
+import com.crowdin.client.fields.model.Field;
+import com.crowdin.client.fields.model.FieldRequest;
+import com.crowdin.client.fields.model.FieldResponseObject;
+import com.crowdin.client.fields.model.FieldResponseObjectList;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class FieldsApi extends CrowdinApi {
+
+    public FieldsApi(Credentials credentials) {
+        super(credentials);
+    }
+
+    public FieldsApi(Credentials credentials, ClientConfig clientConfig) {
+        super(credentials, clientConfig);
+    }
+
+    /**
+     * @param entity Filter fields by entity {@link com.crowdin.client.fields.model.enums.EntityType}
+     * @param search Search fields by slug or name
+     * @param limit  maximum number of items to retrieve (default 25)
+     * @param offset starting offset in the collection (default 0)
+     * @return list of distributions
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.fields.getMany" target="_blank"><b>API Documentation</b></a></li>
+     * </ul>
+     */
+    public ResponseList<Field> listFields(String entity, String search, Integer limit, Integer offset) throws HttpException, HttpBadRequestException {
```

Comment:
Can we use the `EntityType` for the `entity` parameter?

Reply:
In that case do I need to convert that into string lower case or I can just put an Enum?
In my tests mock I pass it as String so they dont match! Or I need to pass it as Enum as well in my mock parameter?

Reply:
What about passing an Enum?

Reply:
You men pass ab Enum in test mock?
If I add Enum in endpoint parameter I need also add Enum in mock parameter. Otherwise they won't match.
I ask it because in API docs entity is in lowercase
Ex: "task"

Reply:
Got you, let's keep it that way as is

Reply:
I've seen at least 2 enums with values here.
So you would like to leave it as String in endpoint parameters?

Reply:
@DecarteAdam have these 2 enums been specified as the parameter type?

Reply:
I don't thik so.
So I leave it as it is? in String format?

Reply:
Let's leave it as it is